packages/terra-brand-footer/src/BrandFooter.jsx

```diff
@@ -9,21 +9,75 @@ const cx = classNames.bind(styles);
 const propTypes = {
  /**
   * An array of navigation links with each element specifiying text, href and target keys with appropriate values.
+  *
+  * **DEPRECATED**: Use `sections` prop instead.
   */
   links: PropTypes.arrayOf(PropTypes.shape({
-    /**
-     * text to be disaplyed as navigational link.
-     */
+   /**
+    * Text to be disaplyed as navigational link.
+    */
     text: PropTypes.string,
-    /**
-     * url of the navigational link.
-     */
+   /**
+    * URL of the navigational link.
+    */
     href: PropTypes.string,
-    /**
-     * attribute to open on same or different tab on clicking the navigational link.
-     */
+   /**
+    * Attribute to open on same or different tab on clicking the navigational link.
+    */
     target: PropTypes.string,
   })),
+ /**
+  * A set of navigation links to be displayed with optional headers. Will override the `links` prop if both are provided.
+  *
+  * ```
+  * Object structured like:
+  * {
+  *   displayVertically: boolean,
+  *   linkGroups: [
+  *     {
+  *       headerText: string,
+  *       links: [
+  *         text: required string,
+  *         href: required string,
+  *         target: string,
+  *       ],
+  *     }
+  *   ],
+  * }
+  * ```
+  */
+  sections: PropTypes.shape({
+   /**
+    * If true link groups will be laid out from top to bottom then left to right
```

Comment:
I am wondering if `isVertical` is more aligned with our boolean prop naming convention.

Additionally, 
>If true link groups will be laid out from top to bottom then left to right

When will it stop laying out from top to bottom and flip to displaying left to right?

Reply:
Looking through this a bit more, I am wondering if it would make more sense to flatten the sections prop to be two props `hasVeriticalLinks` and `linkGroups`. Looking through the logic below it seems we are looping through `sections.linkGroups` or `processedSections.linkGroups` but you don't seem to be doing anything directly with `sections`.